packages/expo-router/src/loaders/utils.ts

```diff
@@ -6,9 +22,8 @@
  * 3. SSR environments
  */
 export async function fetchLoaderModule(routePath: string): Promise<any> {
-  const loaderPath = `/_expo/loaders${routePath}`;
+  const loaderPath = getLoaderModulePath(routePath);
```

Comment:
unrelated to the review: this is a good indicator that we should do better in normalisation here. My assumption is that `/index` is a leaky primitive in some of our route keying, and it's now leaking into route matching for loaders (allowing us to select the route loader more easily when this route is called)

That seems odd to me since we should be able to normalise this on the other end. However, I can see that this is necessary to avoid full matching, which means, this normalisation being done in two other places would save us from having to do it here, which seems less error-prone?